build.sh

```diff
@@ -920,6 +940,21 @@ if [ $WITH_DEPS -gt 0 ]; then
     else
       daphne_msg "No need to build eigen again."
     fi
+    #------------------------------------------------------------------------------
+    # TSL (Template SIMD Library)
+    #------------------------------------------------------------------------------
+    if ! is_dependency_installed "TSL"; then
+        daphne_msg "Install TSL."
+        tsl_generator="${thirdpartyPath}/TSLGenerator"
+        # install TSL requirements
+        pip3 install -r "${tsl_generator}/requirements.txt"
+        tsl_output="${installPrefix}/include/TSL"
+        # python3 ${tsl_generator} --no-workaround-warnings -o ${tsl_output}
+        cmake ${tsl_generator} -B ${tsl_generator}/build -D GENERATOR_OUTPUT_PATH=${tsl_output}
+        dependency_install_success "TSL"
+    else
+        daphne_msg "No need to generate TSL again."
+    fi
```

Comment:
I re-added this to the build script in an attempt to fix the build process. These commands failed in the docker container (I did manually add pip there)